plugin-server/src/worker/ingestion/event-pipeline/5-prepareEventStep.ts

```diff
@@ -11,12 +11,15 @@ export async function prepareEventStep(
     personContainer: LazyPersonContainer
 ): Promise<StepResult> {
     const { ip, site_url, team_id, uuid } = event
+    const invalidTimestampCallback = function (teamId: number) {
+        runner.hub.statsd?.increment('process_event_invalid_timestamp', { teamId: String(teamId) })
```

Comment:
you don't need to pass the teamId either, would argue it's better to use it here directly since you're already using runner directly